tests/unit/test_playback.py

```diff
@@ -0,0 +1,88 @@
+
+
+@pytest.fixture()
+def recording_path() -> Path:
+    return Path(__file__).parent.parent / "assets" / "recording.mkv"
```

Comment:
This file is 88MB. If we add it to the repo it can be long to download for people installing from url. Can we use git LFS? Otherwise, is it possible to have a much smaller recording? Maybe smaller resolution?

Reply:
Upload smaller file: 1.62M
Some problem with LFS for public forks: https://github.com/git-lfs/git-lfs/issues/1906